test/integration/product-import.spec.coffee

```diff
@@ -179,6 +196,9 @@ describe 'Product Importer integration tests', ->
       .then => cleanup(logger, @client.customerGroups, @customerGroup.id)
       .then => cleanup(logger, @client.channels, @channel.id)
       .then => cleanup(logger, @client.categories, @category.id)
+      .then => cleanup(logger, @client.states, @newState.id)
```

Comment:
here we could use something like `cleanAll` which would clean all from the given resource (eg states).